src/tsickle.ts

```diff
@@ -1500,7 +1502,27 @@ class Annotator extends ClosureRewriter {
     this.emit('\n');
     const name = node.name.getText();
 
-    this.emit(`/** @enum {number} */\n`);
+    // Figure out whether this is a string or number enum, by iterating through the enum
+    // members.  Note that TS allows mixed string/number enums also; just translate as {?}.
```

Comment:
what happens if you have a mixed string/non-string enum value?

```ts
enum X {
  A = 1,
  B = 'x',
  C,
}
```

What's `C`? What is it in tsickle?